src/wp-includes/class-requests.php

```diff
@@ -7,7 +7,30 @@
  * Based on concepts from SimplePie_File, RequestCore and WP_Http.
  *
  * @package Requests
+ *
+ * @deprecated 6.2.0
+ */
+
+/*
+ * Integrators who cannot yet upgrade to the PSR-4 class names can silence deprecations
+ * by defining a `REQUESTS_SILENCE_PSR0_DEPRECATIONS` constant and setting it to `true`.
+ * The constant needs to be defined before this class is required.
  */
+if (!defined('REQUESTS_SILENCE_PSR0_DEPRECATIONS') || REQUESTS_SILENCE_PSR0_DEPRECATIONS !== true) {
+	// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
+	trigger_error(
+		'The PSR-0 `Requests_...` class names in the Request library are deprecated.'
+		. ' Switch to the PSR-4 `WpOrg\Requests\...` class names at your earliest convenience.',
+		E_USER_DEPRECATED
+	);
+
+	// Prevent the deprecation notice from being thrown twice.
+	if (!defined('REQUESTS_SILENCE_PSR0_DEPRECATIONS')) {
+		define('REQUESTS_SILENCE_PSR0_DEPRECATIONS', true);
+	}
+}
```

Comment:
Lots of missing whitespace here between parentheses etc. Is this intentional / coming from Requests?

Reply:
Yeah it's coming from Requests and this file is excluded from `phpcs.xml.dist` - [See here](https://github.com/WordPress/wordpress-develop/blob/trunk/phpcs.xml.dist#L116).